lib/jsdom/living/helpers/style-rules.js

```diff
@@ -235,6 +237,29 @@ function getComputedValue(element, property) {
     const specifiedValue = getSpecifiedValue(element, property);
 
     // https://drafts.csswg.org/css-color-4/#resolving-other-colors
+    let value = "";
+    if (specifiedValue === "initial") {
+      value = initial;
+    } else if (specifiedValue === "inherit") {
+      if (element.parentElement) {
+        return getComputedValue(element.parentElement, property);
+      }
+      value = initial;
+    } else {
+      value = asciiLowercase(specifiedValue);
+    }
+    if (systemColors.has(value) || deprecatedAliases.has(value)) {
+      let key = value;
+      if (deprecatedAliases.has(value)) {
+        key = deprecatedAliases.get(value);
+      }
+      const { light, dark } = systemColors.get(key);
+      const colorScheme = getCascadedPropertyValue(element, "color-scheme");
+      if (colorScheme === "dark") {
+        return dark;
+      }
+      return light;
+    }
     if (specifiedValue === "currentcolor") {
```

Comment:
As a style issue, can we move this to be right after the other `specifiedValue` tests? It should not make any functional difference, but I like the flow of handle special `specifiedValue` => handle system color specified values => return computed value.